graveler/committed/sstable/manager.go

```diff
@@ -0,0 +1,96 @@
+package sstable
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"hash"
+
+	"github.com/treeverse/lakefs/logging"
+
+	"github.com/treeverse/lakefs/pyramid"
+
+	"github.com/treeverse/lakefs/graveler"
+)
+
+type PebbleSSTableManager struct {
+	cache  cache
+	fs     pyramid.FS
+	logger logging.Logger
+	hash   hash.Hash
+}
+
+const sstableTierFSNamespace = "sstables"
+
+func NewPebbleSSTableManager(cache cache, fs pyramid.FS) Manager {
```

Comment:
cso this implements `sstable.Manager`?
I have to say, looking at the interface itself, it is *completely* decoupled from the fact that the underlying format is an sstable! that's a good thing. I think maybe it could even be defined one layer up, as part of `committed`. This way, trees work against a truely generic interface dealing with storage and are not bound to a specific format (as a matter of fact, this is alreay the case, it's only the package name of `sstable.Manager` that implies any sort of coupling..)

@itaiad200  wdyt?